Sources/AppcuesKit/Appcues.swift

```diff
@@ -139,8 +139,8 @@ public class Appcues: NSObject {
     /// the `identify` call.  This will cause the SDK to begin tracking activity and checking for
     /// qualified content.
     @objc
-    public func anonymous(properties: [String: Any]? = nil) {
-        identify(isAnonymous: true, userID: config.anonymousIDFactory(), properties: properties)
+    public func anonymous() {
+        identify(isAnonymous: true, userID: config.anonymousIDFactory(), properties: nil)
```

Comment:
I don't think it's necessary for this change since the error "`Argument passed to call that takes no arguments`" is good enough, but we could do this to generate a slightly nicer call site error:

```swift
@available(*, unavailable, message: "properties are no longer supported for anonymous users.")
public func anonymous(properties: [String: Any]? = nil) {
    // removed
}
```

<img width="1605" alt="Screenshot 2023-02-22 at 9 28 43 AM" src="https://user-images.githubusercontent.com/845681/220652094-966df1f8-e6ad-4252-8d87-0241057cf0b7.png">

Good references to keep in mind for future breaking changes:

- https://www.hackingwithswift.com/example-code/language/how-to-use-available-to-deprecate-old-apis
- https://nshipster.com/available/

Reply:
good idea! thank you